src/function/memo.rs

```diff
@@ -312,19 +305,23 @@ impl<V> Memo<V> {
     }
 }
 
-impl<V: Send + Sync + Any> crate::table::memo::Memo for Memo<V> {
+impl<C: Configuration> crate::table::memo::Memo for Memo<'static, C>
```

Comment:
Given that memo now has access to the `Configuration`. Doesn't that mean that we could get the debug name too?

Reply:
We can also do this in a separate PR. It was more a thought that I had. As you preferred